inspire_dojson/common/rules.py

```diff
@@ -325,14 +282,23 @@ def public_notes_680(self, key, value):
 @jobs.over('urls', '^8564.')
 @journals.over('urls', '^8564.')
 def urls(self, key, value):
+    def _is_internal_url(url):
+        base = urllib.parse.urlparse(current_app.config['LEGACY_BASE_URL'])
```

Comment:
@david-caro @kaplun not sure about using a config variable here, should I rather hardcode `https?://inspirehep.net`? If this var has the same value as the one for robotubload, it will break on qa because the records from the dump/prodsync have http://inspirehep.net hardcoded in them in `8564`.